### DISCUSSION TITLE
What is the action for sigint (ctrl-c)?

### DISCUSSION THREAD

**Discussion by haydonryan** [ghostty-org/ghostty#3516] [Q&A] [answered]
I'm remapping copy and paste as follows:
```
keybind = ctrl+c=copy_to_clipboard
keybind = ctrl+v=paste_from_clipboard
```

But can't seem to find how to map sigint (the old control-c) to ctrl+shift+c.  The docs aren't helpful (it's probably a skill issue on my part).

Anyone able to help? Thanks!

**Comment by jcollie** [comment, marked as answer]
There's nothing special about control-c:

`keybind =ctrl+k=text:\x03`

**Reply by haydonryan** [reply]
Awesome thankyou.

**Comment by haydonryan** [comment]
I would also add - It would be great to have this as an alias - rather than text:\x03
